index.html

```diff
@@ -696,6 +699,212 @@ <h1 class="display-1">RPG Tools</h1>
                 </div>
                 <!-- /Coriolis Weapon Reference -->
 
+                <!-- Coriolis Armor Reference -->
+                <div class="coriolis-armor">
+                    <div class="row mb-3">
+                        <div class="col-lg-12">
+                            <button class="btn btn-dark mb-2">Jump to</button>
+                            <button class="btn btn-primary mb-2 to-coriolis-armor">Armor</button>
+                            <button class="btn btn-primary mb-2 to-coriolis-exos">Exos</button>
+                            <button class="btn btn-primary mb-2 to-coriolis-shields">Shields</button>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-lg-12">
+                            <table class="table table-sm table-sm-text table-hover coriolis-armor">
+                                <thead class="thead-dark">
+                                    <tr>
+                                        <th colspan="6">Armor</th>
+                                    </tr>
+                                    <tr>
+                                        <th style="width: 20%">Name</th>
```

Comment:
Please change the fixed-fraction, inline styles to Bootstrap classes for column sizes. They're all based on a 12-column grid so col-lg-1 through col-lg-12 will give you the flexibility as long as the sum adds to 12. This will enable mobile scaling and stylesheet updates.

Reply:
Well, this ended up being much more of a pain then expected. In order to make this work, you have to use the `col-lg-...` classes on every `th` and `td`, additionally, any `tr` that uses those classes also has to have the `d-flex` class to make it work.